### ISSUE TITLE
Should warn the player about item decay

### ISSUE THREAD

**Issue by mafagafogigante** [mafagafogigante/dungeon#380]
The inventory should keep track of what rotted so that we can tell the player why things aren't there anymore.

**Comment by UnknownNPC** [comment]
@mafagafogigante also, foods/fruits with less quality, should restore less heals. I faced with it in the another game, but I can remember it name. It sounds like another task. :)

**Comment by mafagafogigante** [comment]
@UnknownNPC I'm not sure about what do you mean with that. Do you mean fruits that have already rotten a bit? All fruits are equally "good" from the start, right?

**Comment by mafagafogigante** [comment]
@UnknownNPC if you are interested in adding this feature (a message queue to the inventory), please let me know.

**Comment by UnknownNPC** [comment]
>All fruits are equally "good" from the start, right?

Yeah! Something like, `Fresh/Good` (100% restore) on the start, after 5 game days it changes status to ` Slightly rotten` (50% restore), after some more days it changes status to `Rotten` (10% restore).
It's like thinking out loud, after your first comment reading.

>if you are interested in adding this feature (a message queue to the inventory), please let me know.
Sure, It can be assigned on me.

**Comment by mafagafogigante** [comment]
@UnknownNPC I will think about the "partial decay" aspect of food some more and maybe create an issue for it.

What is expected from this issue is something that works this way:

Say that an update a few items were rotten (and therefore completely removed from the Hero inventory), the next time I issue "items" I want to see lines such as

"You notice [item name] rotted in your inventory."

for each item that rotted. If you have any questions, just ask them.
